packages/web-pixel-ui-extensions/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@shopify/web-pixel-ui-extensions",
+  "description": "Provides tools to create Web Pixel extension",
+  "version": "0.0.1",
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "sideEffects": false,
+  "license": "MIT",
+  "main": "index.js",
+  "module": "index.mjs",
+  "esnext": "index.esnext",
+  "types": "index.d.ts",
+  "exports": {
+    ".": {
+      "esnext": "./index.esnext",
+      "import": "./index.mjs",
+      "require": "./index.js"
+    },
+    "./": "./"
```

Comment:
You can remove this, it's deprecated and the other packages shouldn't have it either